packages/react-native-gesture-handler/src/web/handlers/NativeViewGestureHandler.ts

```diff
@@ -40,9 +45,11 @@ export default class NativeViewGestureHandler extends GestureHandler {
     if (this.config.disallowInterruption !== undefined) {
       this.disallowInterruption = this.config.disallowInterruption;
     }
-
-    const view = this.delegate.view as HTMLElement;
-    this.restoreViewStyles(view);
+    if (this.delegate.initialized) {
+      // this function is called on handler creation, which happens before initializing delegate
+      const view = this.delegate.view as HTMLElement;
+      this.restoreViewStyles(view);
+    }
```

Comment:
Doesn't this break the native gesture somewhat? If the view isn't set when `init` is called, `restoreViewStyles` wouldn't be called at all, no?